runtime/doc/lsp.txt

```diff
@@ -60,6 +60,14 @@ options are not restored when the LSP client is stopped or detached.
   - To opt out of this use |gw| instead of gq, or set 'formatexpr' on LspAttach.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
+- *crr* is mapped in Normal mode to |vim.lsp.buf.rename()| if no existing
```

Comment:
I think we should also document on how to modify/remove these if desired.

Reply:
Already mentioned just below this, but I updated the example to include the new default mappings too.